### PR TITLE
feat(route53): add classic elb target support

### LINKED ISSUE #3306: Use classic load balancer (ELB) as a Route53 target
* **I'm submitting a ...**
  - [ ] :beetle: bug report
  - [ ] :rocket: feature request
  - [X] :books: construct library gap
  - [ ] :phone: security issue or vulnerability => Please see [policy](https://github.com/aws/aws-cdk/security/policy)
  - [ ] :question: support request => Please see note at the top of this template.

* **What is the expected behavior (or behavior of feature suggested)?**

The Classic load balancer (ELB) is currently not supported as a Route53 Target.

* **What is the motivation / use case for changing the behavior or adding this feature?**

I'm using the Kubernetes Nginx Ingress, which currently only supports the use of the classic load balancer.

* **Please tell us about your environment:**

  - CDK CLI Version: 1.0.0
  - Module Version: 1.0.0
  - OS: [all]
  - Language: [all ]

### PR DESCRIPTION
**Pull request by nmussy**
* adds Route 53 ClassicLoadBalancerTarget class
* updated README accordingly

Fixes #3306

----

Please read the [contribution guidelines](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md) and follow the pull-request checklist.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-route53-targets/lib/classic-load-balancer-target.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import elb = require('@aws-cdk/aws-elasticloadbalancing');
+import route53 = require('@aws-cdk/aws-route53');
+
+/**
+ * Use a classic ELB as an alias record target
+ */
+export class ClassicLoadBalancerTarget implements route53.IAliasRecordTarget {
+  constructor(private readonly loadBalancer: elb.LoadBalancer) {
```
Comment (nmussy) [inline packages/@aws-cdk/aws-route53-targets/lib/classic-load-balancer-target.ts]: I ended up creating a separate class, please tell me if you'd prefer having an union. The only issue is that the `hotsedZoneId` property is mapped differently in v1 versus v2,`loadBalancerCanonicalHostedZoneNameId`  and `loadBalancerCanonicalHostedZoneId` respectively

File: packages/@aws-cdk/aws-route53-targets/README.md [hunk 2]
```diff
@@ -33,5 +33,13 @@ This library contains Route53 Alias Record targets for:
     // or - route53.RecordTarget.fromAlias(new alias.ApiGatewayDomainName(domainName)),
   });
   ```
+* Classic load balancers
+  ```ts
+  new route53.ARecord(this, 'AliasRecord', {
+    zone,
+    target: route53.RecordTarget.fromAlias(new alias.LoadBalancerTarget(elb)),
```
Comment (jogold) [inline packages/@aws-cdk/aws-route53-targets/README.md]:
```suggestion
    target: route53.RecordTarget.fromAlias(new alias.ClassicLoadBalancerTarget(elb)),
```

Comment (nmussy) [reply packages/@aws-cdk/aws-route53-targets/README.md]: Thanks 👍

File: packages/@aws-cdk/aws-route53-targets/lib/index.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
 export * from './api-gateway-domain-name';
 export * from './cloudfront-target';
 export * from './load-balancer-target';
+export * from './classic-load-balancer-target';
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-route53-targets/lib/index.ts]: can we make this alphabetical? - It's easier to search and find if there isn't a strong reason to append at the end.

Comment (nmussy) [reply packages/@aws-cdk/aws-route53-targets/lib/index.ts]: Done 👍

**Pull request merged**